resources/src/main/java/org/robolectric/res/android/ResTable.java

```diff
@@ -51,9 +51,6 @@
 @SuppressWarnings("NewApi")
 public class ResTable {
 
-  private static final int IDMAP_MAGIC             = 0x504D4449;
```

Comment:
Those two fields maybe useful for overlay mechanism provided by OverlayManagerSevice based on `idmap`, although we don't support it yet. Maybe we can keep those two fields.

Reply:
Okay, I'll suppress it as well.

but @utzcoz there is a little interesting debate on the topic regarding if something we are not going to use now but maybe in the future, some devs keep it, and some devs clean it and refer to it through commit later whats you're take on this? I'm eager to know

I prefer keeping code clean as it's a liability moreover the way I think and we can anytime add necessary things at a later point when we actually need actually.

Reply:
It depends on. In many occasion, I would like to remove unused fields, methods and other code. But I am not comfortable to remove unused constants, used for special magic number, in new resource implementation, because it maybe used someday. And it's hard for us to find it. Anyway, we have git log, and we can use git log to find it or we can search AOSP source code to find the value of `IDMAP_MAGIC` if we need to support something related to it.